crates/iota-light-client/Cargo.toml

```diff
@@ -5,35 +5,54 @@ authors = ["IOTA Foundation <info@iota.org>"]
 edition = "2021"
 license = "Apache-2.0"
 publish = false
+description = "A light client for the IOTA blockchain"
 
 [lib]
 path = "src/lib.rs"
 
 [[bin]]
 name = "iota-light-client"
-path = "src/bin/light_client.rs"
+path = "src/main.rs"
+
+[[bin]]
+name = "generate-chk-snapshots"
+path = "src/bin/generate_chk_snapshots.rs"
 
 [dependencies]
 # external dependencies
 anyhow.workspace = true
 async-trait.workspace = true
 bcs.workspace = true
+bin-version.workspace = true
 clap.workspace = true
 env_logger = "0.11.5"
+futures.workspace = true
+getset = "0.1"
 log = "0.4.22"
 object_store.workspace = true
+prometheus.workspace = true
 reqwest.workspace = true
+roaring.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 serde_yaml.workspace = true
+telemetry-subscribers.workspace = true
+tempfile = "3.8.0"
 tokio = { workspace = true, features = ["full"] }
+tracing.workspace = true
 url.workspace = true
 
 # internal dependencies
+iota-archival.workspace = true
 iota-config.workspace = true
+iota-data-ingestion-core.workspace = true
+iota-graphql-rpc-client.workspace = true
 iota-json-rpc-types.workspace = true
 iota-package-resolver.workspace = true
 iota-rest-api.workspace = true
 iota-sdk.workspace = true
 iota-types.workspace = true
 move-core-types.workspace = true
+
+[dev-dependencies]
+tempfile.workspace = true
```

Comment:
probably don't need it here if you have it in dependencies already?

Reply:
it's the other way around actually (was probably due to some testing)

https://github.com/iotaledger/iota/pull/6135/commits/173fc7c0428bc6d8389e31a8bda944c3a7c95da3